src/Workspaces/Core/Portable/Formatting/Engine/TokenStream.cs

```diff
@@ -565,11 +609,10 @@ private sealed class TokenOrderComparer : IComparer<SyntaxToken>
         {
             public static readonly TokenOrderComparer Instance = new TokenOrderComparer();
 
-            private TokenOrderComparer() { }
```

Comment:
this private constructor is intentional.  it makes sure that people only use the static member.  cna you add this back in?  Thanks!

Reply:
Restored. Previously was removed as you can't have own parameterless constructor on structs.